internal/impl/aws/input_kinesis.go

```diff
@@ -692,6 +648,8 @@ func (k *kinesisReader) runExplicitShards() {
 	}
 }
 
+// This function attempts to try and claim shards only if the number of shards already claimed is less than the max number of shards
```

Comment:
```suggestion
// maxShardRunBalancedConsumer attempts to try and claim shards only if the number of shards already claimed is less than the max number of shards
```

Convention is to start the doc blocks with the name of the function/struct you are leaving the comment for

Reply:
Gotcha thanks!